Resources/Prototypes/Entities/Structures/Furniture/chairs.yml

```diff
@@ -181,6 +181,110 @@
     graph: Seat
     node: chairComfy
 
+- type: entity
+  name: black comfy chair
+  id: BlackComfyChair
+  parent: ChairBase
+  description: It looks comfy.
+  components:
+  - type: Sprite
+    sprite: Structures/Furniture/comfy_chair.rsi
+    state: comfy-black
+  - type: Construction
+    graph: Seat
+    node: blackChairComfy
```

Comment:
Please use sprite layer variants instead of creating one sprite for each color. You can look at screwdrivers for how color layers work.